bin/setup_service_account_permissions

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -e
+project=$1
+service_account=$2
+custom_role_name="CustomStorageRole"
+function usage {
+   echo Usage: setup_service_account_permissions project service_account
```

Comment:
maybe just set_roles?  The long form is a bit unwieldy (and it will get in the way of directory listings, etc...)